src/lib/y2storage/lvm_pv.rb

```diff
@@ -26,7 +26,7 @@ class LvmPv < Device
     #
     # @return [BlkDevice]
     def plain_blk_device
-      blk_device.encryption? ? blk_device.blk_device : blk_device
+      blk_device.is_a?(Encryption) ? blk_device.blk_device : blk_device
```

Comment:
just note to discussion we have about it ( so not need for action, just showing how ask can be turn into say). For this case it make sense to have method like `plain_device` that return itself for base class and for Encryption it return blk_device

so code can look like
```
class BlkDevice
  def plain_device
     self
  end
end

class Encryption
  def plain_device
     blk_device.plain_device # in case if two encryption is possible
  end
end
```

and then this code is not needed, as you call `lvm_pv.blk_device.plain_device` when need it as plain and for above case
```
def plain_blk_devices
  blk_devices.map(&:plain_device)
end
```
or removing, as this call is already easy enough to use it when makes sense.